code/src/utest/plugin/function/TestUtilities.java

```diff
@@ -21,40 +21,39 @@
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 
-import junit.framework.TestCase;
 import pcgen.base.formula.parse.FormulaParser;
 import pcgen.base.formula.parse.ParseException;
 import pcgen.base.formula.parse.SimpleNode;
 
-public class TestUtilities
+import static org.junit.Assert.*;
+
+public final class TestUtilities
 {
 
-	public static SimpleNode doParse(String formula)
+	private TestUtilities()
+	{
+	}
+
+	static SimpleNode doParse(String formula)
 	{
 		try
 		{
 			return new FormulaParser(new StringReader(formula)).query();
 		}
 		catch (ParseException e)
 		{
-			TestCase
-				.fail("Encountered Unexpected Exception: " + e.getMessage());
+			fail("Encountered Unexpected Exception: " + e.getMessage());
 			return null;
 		}
 	}
 
-	public static final double SMALL_ERROR = Math.pow(10, -10);
+	private static final double SMALL_ERROR = Math.pow(10, -10);
 
-	public static boolean doubleEqual(double d1, double d2, double delta)
+	static boolean doubleEqual(double d1, double d2)
 	{
-		if (delta < 0)
-		{
-			throw new IllegalArgumentException(
-				"Delta for doubleEqual cannot be < 0: " + delta);
-		}
 		double diff = d1 - d2;
-		return ((diff >= 0) && (diff < delta))
-			|| ((diff < 0) && (diff > -delta));
+		return ((diff >= 0) && (diff < TestUtilities.SMALL_ERROR))
```

Comment:
Fundamentally I disagree with this merge - it is forcing a specific delta that the utility class claims it knows better than the caller.  That will not be universally true, and therefore this should not be changed.  I'll merge anyway because I really am less concerned about our test class structure, but for awareness...